.stylelintrc

```diff
@@ -2,6 +2,9 @@
     "extends": ["stylelint-config-standard"],
     "customSyntax": "postcss-less",
     "rules": {
+        "string-quotes": null,
```

Comment:
this rule is already deprecated, why do you use this?
https://stylelint.io/user-guide/rules/string-quotes/
I mean what a case you tried to avoid?